generators/python/src/fern_python/snippet/snippet_template_factory.py

```diff
@@ -100,7 +100,7 @@ def _expression_to_snippet_str(
         self,
         expr: AST.Expression,
     ) -> str:
-        snippet = SourceFileFactory.create_snippet()
+        snippet = SourceFileFactory.create_snippet(should_format=True)
```

Comment:
These low key could respect the flag too since we have the resolve with formatting call within the template resolver lib